tests/test_distributions.py

```diff
@@ -348,6 +362,8 @@ def test_rvs_test_point_are_valid(tf_seed, distribution_conditions):
     dist_class = getattr(pm, distribution_name)
     dist = dist_class(name=distribution_name, **conditions)
     test_value = dist.test_value
+    if distribution_name in ["Flat", "HalfFlat"]:
+        pytest.skip("Flat and HalfFlat distributions don't support sampling.")
```

Comment:
We shouldn't skip this test. In the case of the `Flat` and `HalfFlat`, instead of comparing the `test_value.shape` against the `test_sample.shape`, we should compare it against the `sample_shape + batch_shape + event_shape` that we expect to get.